diff -ruN a/avogadrolibs/python/CMakeLists.txt b/avogadrolibs/python/CMakeLists.txt
--- a/avogadrolibs/python/CMakeLists.txt	2023-10-26 09:59:34.000000000 +0200
+++ b/avogadrolibs/python/CMakeLists.txt	2023-10-27 00:07:43.788724322 +0200
@@ -17,7 +17,7 @@
 set(_python_module_install_dir "avogadro")
 # SKBUILD is set for binary wheel
 if (NOT SKBUILD)
-  set(_python_module_install_dir "${Python_STDARCH}/avogadro")
+  set(_python_module_install_dir "/app/lib/avogadro")
 endif()
 
 set(CMAKE_MODULE_LINKER_FLAGS "")
